src/Destructurama.ByIgnoring/ByIgnoring/IgnoredPropertyExpressionExtensions.cs

```diff
@@ -17,11 +17,17 @@
 
 namespace Destructurama.ByIgnoring
 {
-    static class IgnoredPropertyExpressionExtensions
+    /// <summary>
+    /// Extension methods used to obtain property names.
+    /// </summary>
+    public static class IgnoredPropertyExpressionExtensions
```

Comment:
Whe public?

Reply:
I made it public so that the test project could access the method (`ByIgnoreWhereTestCases.cs:125`). I'm not a huge fan of this to be honest (it was an effort to reduce what are essentially duplicate test case definitions), so if it's a deal-breaker I can come up with another solution and make this internal again.

Reply:
InternalsVisibleTo is a well known way for testing.

Reply:
fixed in #25